src/frontend/src/lib/components/tokens/Tokens.svelte

```diff
@@ -17,9 +17,9 @@
 
 	{#if $authSignedIn}
 		<TokensSignedIn />
+
+		<ManageTokensButton />
```

Comment:
Does it needs a `fade` effect?

Reply:
Good one! Added `fade` transition (same as token list)


https://github.com/user-attachments/assets/cd8d3fe4-3ab5-4dc1-9642-7e48f1812ba4